index.html

```diff
@@ -12,11 +12,11 @@
         font-size: 12px;
       }
 	  
-	  #Edit {
-	    position: relative;
-		top: 620px; /*Relative to height of 'bar-chart-large'*/	
-		left: 20px;
-	  }
+	    #Edit {
```

Comment:
You have tab characters here. This project is using 2-space characters for indenting. Please replace the tabs with appropriate spaces.